packages/fxa-settings/src/styles/unit-row.scss

```diff
@@ -7,6 +7,8 @@
 
   &-header {
     @apply font-header w-full mb-1;
+    min-width: 165px;
+
```

Comment:
The New Settings Style Guide: "... it's important to keep in mind that our CSS system for FxA styles increase in units of 4px. If a design reflects 17px, it's generally safe to go with the closest value divisible by 4, 16px (which is equivalent to 1rem) ..."

Any reason we cannot use a `rem` here?

On my computer, the row header for the other rows were less than a pixel wider than the one for the enabled Two Step Auth row.